lua/lib/renderer.lua

```diff
@@ -160,13 +160,22 @@ local special = {
   ["readme.md"] = true,
 }
 
+local root_folder_modifier = vim.g.lua_tree_root_folder_modifier or ':~'
+
 local function update_draw_data(tree, depth, markers)
   if tree.cwd and tree.cwd ~= '/' then
-    table.insert(lines, "..")
-    table.insert(hl, {'LuaTreeFolderName', index, 0, 2})
+    table.insert(lines, ".. (up a dir)")
+    table.insert(hl, {'LuaTreeFolderName', index, 0, 13})
```

Comment:
should we add this `up a dir` thing ? I guess people using unix already know about that ^^

Reply:
yep this `up a dir` takes too much space on small screens, my font for directories is bold and it does not look very good ^^

Reply:
could you also switch the `..` and the path so the path is at the top ?

Reply:
@kyazdani42 so you mean to merge first two lines into one?
In my example, it would look like this?
```
~/github/nvim-tree/..
doc/
lua/
...
```

Reply:
I just did that. It is indeed better:

![screenshot](https://i.imgur.com/6nOh6yW.png)